slsDetectorServers/jungfrauDetectorServer/slsDetectorServer_defs.h

```diff
@@ -67,9 +67,9 @@ enum DACINDEX {
 #define NSPECIALDACS                         (3)
 #define SPECIALDACINDEX                      {J_VREF_PRECH, J_VREF_DS, J_VREF_COMP};
 #define SPECIAL_DEFAULT_DYNAMIC_GAIN_VALS                                      \
-    { 1000, 500, 400 }
+    { 1450, 480, 420 }
```

Comment:
I didn't expect this to change in this PR. Which settings are new and which are old?

Reply:
Yes, this and G2 are the things I am adding in here.